index.html

```diff
@@ -0,0 +1,178 @@
+                <h3 class="location">@ Filmcity Goregaon Event Hall</h3>
+            </div>
+        </div>
+    </section>
+    <!-- Main Program -->
+    <section>
+        <div class="metalBackground">
+            <div class="metalBackgroundHolder">
+                <div class="main">
+                    <div class="titleContainer">
+                        <h2 class="mainTitle">Main Program</h2>
+                        <div class="line"></div>
+                    </div>
+                    <ul class="mainContent">
+                        <li class="bands">
+                            <span class="material-icons-outlined icon">headphones</span>
+                            <h3 class="band">JBL</h3>
+                            <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Magni impedit iusto quam sapiente necessitatibus.</p>
+                        </li>
+                        <li class="bands">
+                            <span class="material-icons-outlined icon">graphic_eq</span>
+                            <h3 class="band">INTEL</h3>
+                            <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Magni impedit iusto quam sapiente necessitatibus.</p>
+                        </li>
+                        <li class="bands">
+                            <span class="material-icons-outlined icon">audiotrack</span>
+                            <h3 class="band">SONI MUSIC</h3>
+                            <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Magni impedit iusto quam sapiente necessitatibus.</p>
+                        </li>
+                        <li class="bands">
+                            <span class="material-icons-outlined icon">speaker</span>
+                            <h3 class="band">DC</h3>
+                            <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Magni impedit iusto quam sapiente necessitatibus.</p>
+                        </li>
+                        <li class="bands">
+                            <span class="material-icons-outlined icon">earbuds</span>
+                            <h3 class="band">BOAT</h3>
+                            <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Magni impedit iusto quam sapiente necessitatibus.</p>
+                        </li>
+                    </ul>
+                    <a class="mainButton" href="#">Sony Music Fall</a>
+                </div>
+            </div>
+        </div>
+    </section>
+    <!-- section to locate the tags created from javaScript -->
+    <section class="information" >
+        <div class="divTitle">
+            <h2>Singers and location inforamtion</h2>
+            <div>
+        </div>
+    </section>
+    <!-- Footer section -->
+    <footer>
+        <div class="titleContainer">
+            <h2 class="mainTitle fooTitle"> OurPartners</h2>
+            <div class="line"></div>
+        </div>
+    </footer>
```

Comment:
- [ ] Kindly add some padding on the image so that it can have space around it like it is in the given design
- [ ] Also add some padding around the `div` so as to achieve the positioning in the given design

Your design
![image](https://user-images.githubusercontent.com/81932478/205454514-fe77ea15-a401-4017-94b0-127329c4f9c4.png)

GIven design
![image](https://user-images.githubusercontent.com/81932478/205454520-2b71ef35-d5fb-48bf-9935-7e78c1fa31b0.png)